static/css/styles.css

```diff
@@ -0,0 +1,8 @@
+body {
+    margin-top: 20px;
+}
+
```

Comment:
Add visible styles (e.g. change the background color) here and update the screenshot to indicate that static files are served correctly